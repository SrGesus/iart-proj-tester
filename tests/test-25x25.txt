VC	LV	VE	VC	LV	LH	LV	BD	LV	LH	VC	VC	VD	VD	LV	LH	FD	VC	LH	LV	LV	LH	LH	BE	VE
BC	VE	LV	LH	VE	LV	FC	VB	LV	VD	VC	VC	LV	LH	VD	LV	VB	LV	VB	LV	LV	VB	VB	VD	LV
LV	LV	FC	LV	BE	LV	BE	VE	VE	VC	VB	FB	VC	BC	VB	VC	VD	VC	LH	FE	VD	LV	VE	VE	LV
FD	LH	VB	VB	LH	VD	VB	LH	VB	VB	BD	LH	LV	VB	VE	VC	VB	LV	BD	LV	VB	VB	VD	LV	LV
VD	VD	LV	FD	LH	LV	FC	VC	VD	LH	LH	FC	BC	VC	VE	VB	FD	VC	LV	VB	VD	VB	LV	LH	LH
LV	VB	VD	LH	LV	LV	BE	VD	LH	LV	VE	VB	LH	BE	VB	LH	VB	VB	VD	BD	VC	VD	VC	LH	LV
LH	VC	VC	VD	VB	LH	LV	FC	LV	LH	FC	BB	FB	LH	LH	LV	LH	VD	VE	FE	VC	LH	LV	VC	LH
BC	VE	VD	LV	VB	LV	VE	LH	BD	VE	VC	LH	VB	VB	LH	VE	VB	LV	VC	BE	LV	VC	LH	VD	FC
FD	VE	BB	FC	LV	LH	VE	VC	VE	VC	LV	LH	VB	VD	FC	VB	VE	VB	VB	LV	VD	VB	FC	VB	VC
VE	LV	VD	VE	VB	VD	VE	VB	VB	FB	LH	VC	LV	BC	VD	VE	VD	VC	VD	VB	VD	VE	LH	VC	VC
LH	VB	VC	LV	VE	LV	LV	VE	BD	VE	VC	LV	VC	FD	LH	VC	VC	VC	BE	FC	BD	BC	VD	VE	VE
LH	LH	VD	BC	VC	VB	VD	LV	FD	BD	LH	VD	VC	VE	LH	LH	VD	VD	BD	LH	VE	LH	VC	VB	LH
BE	VB	VC	LH	VB	FE	VC	BE	LH	VD	FB	BC	FB	VE	VB	LH	FE	LV	FB	VD	VC	FE	LH	VE	BE
VD	VC	VD	VD	VC	LH	LV	LH	LH	VE	VD	BD	BD	FC	VB	VB	LV	VD	LH	VE	VE	VD	BB	VC	FB
FB	VC	VC	VD	FB	BE	BE	LV	FB	VE	LV	VB	FE	VC	VD	VD	BE	FB	VE	VD	LV	VC	FB	VC	VC
VD	LV	VE	FB	BD	VE	VC	BB	LV	BC	VB	VD	LH	VB	FC	VE	BD	VE	LH	VD	VD	VC	BB	LV	BB
BE	VE	VE	VB	FC	VE	VC	VE	FC	LH	LV	VD	BC	LH	LH	VE	LH	LV	VD	VB	LH	FD	VE	VD	LH
LV	LV	LV	VC	LV	BD	LH	VE	LH	VE	LV	LH	VD	LV	VB	LH	LV	VB	LH	VD	VD	LH	LH	VE	LV
FE	VD	VE	VC	VD	FE	LH	BE	LV	VE	FB	VE	VE	VC	VC	LH	VE	VE	VE	LV	LV	LV	VB	VB	VB
VB	LH	LV	VC	VD	LH	VE	LV	FD	VB	LV	LH	VB	VB	VC	VD	LV	BE	VC	BE	FD	VC	BD	VD	VE
VC	VD	FC	BC	LH	LV	VD	LH	BE	LV	BB	LH	LH	VD	VD	LH	VC	VC	VB	LH	VC	VB	BD	FB	LH
FB	VC	VE	FC	FD	VD	BD	VC	VC	VC	VC	LV	VE	VD	BD	FC	VB	VD	LV	LV	LH	VC	VD	VD	VC
BB	FE	VD	LV	LV	VC	LH	VC	VD	FC	VB	VD	LH	VE	VE	VB	LH	VC	LV	LV	LH	VB	VC	VC	VD
LH	VC	LH	LH	LH	LV	VC	LV	LH	VC	VE	VC	VB	LH	VE	VC	VC	VC	FB	LH	VD	VD	FD	VB	VD
VB	BE	LH	LH	LV	LV	LV	VE	VC	VD	FC	LH	LH	VE	VE	LH	VB	VB	LV	BC	FB	VD	LH	BE	FC
