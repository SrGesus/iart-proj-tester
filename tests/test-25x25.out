VB	LH	VE	VB	LH	LH	LH	BB	LH	LH	VE	VB	VE	VB	LH	LH	FE	VB	LH	LH	LH	LH	LH	BB	VE
BD	VE	LV	LV	VB	LH	FE	VD	LH	VE	VD	VC	LV	LV	VB	LH	VE	LV	VB	LH	LH	VE	VB	VC	LV
LV	LV	FC	LV	BD	LH	BB	VE	VB	VC	VB	FE	VD	BE	VD	VE	VD	VC	LV	FD	VE	LV	VD	VE	LV
FC	LV	VB	VC	LV	VB	VC	LV	VD	VE	BD	LH	LH	VC	VB	VC	VB	LH	BC	LH	VC	VD	VE	LV	LV
VB	VC	LV	FB	LV	LV	FB	VD	VE	LV	LV	FD	BB	VE	VD	VE	FC	VB	LH	VE	VB	VE	LV	LV	LV
LV	VB	VC	LV	LV	LV	BD	VE	LV	LV	VD	VE	LV	BD	VE	LV	VB	VC	VB	BC	VC	VD	VC	LV	LV
LV	VD	VE	VD	VC	LV	LV	FC	LV	LV	FD	BE	FC	LV	LV	LV	LV	VB	VC	FB	VB	LH	LH	VC	LV
BD	VE	VD	LH	VE	LV	VD	LH	BE	VD	VE	LV	VB	VC	LV	VD	VC	LV	VB	BE	LV	VB	LH	VE	FC
FC	VD	BB	FE	LV	LV	VB	VE	VD	VE	LV	LV	VD	VE	FC	VB	VE	VD	VC	LV	VD	VC	FB	VD	VE
VB	LH	VC	VB	VC	VD	VC	VD	VE	FC	LV	VD	LH	BE	VB	VC	VD	VE	VB	VC	VB	VE	LV	VB	VC
LV	VB	VE	LV	VB	LH	LH	VE	BD	VE	VD	LH	VE	FC	LV	VB	VE	VD	BE	FD	BE	BD	VC	VD	VE
LV	LV	VD	BC	VC	VB	VE	LV	FC	BD	LH	VE	VD	VE	LV	LV	VD	VE	BD	LH	VC	LV	VB	VE	LV
BD	VC	VB	LH	VE	FC	VD	BC	LH	VC	FB	BD	FE	VD	VC	LV	FB	LV	FC	VB	VE	FC	LV	VD	BE
VD	VE	VD	VE	VD	LH	LH	LH	LH	VE	VD	BC	BB	FE	VB	VC	LV	VD	LH	VC	VD	VE	BD	VE	FC
FD	VC	VB	VC	FD	BB	BB	LH	FE	VD	LH	VE	FC	VB	VC	VB	BE	FD	VE	VB	LH	VC	FC	VD	VE
VB	LH	VC	FD	BB	VC	VD	BB	LH	BB	VE	VD	LH	VC	FD	VC	BD	VE	LV	VD	VE	VB	BB	LH	BE
BD	VE	VB	VE	FC	VB	VE	VD	FE	LV	LV	VB	BB	LH	LH	VE	LV	LV	VD	VE	LV	FC	VD	VE	LV
LV	LV	LV	VD	LH	BE	LV	VB	LH	VC	LV	LV	VD	LH	VE	LV	LV	VD	LH	VC	VD	LH	LH	VC	LV
FC	VD	VC	VB	VE	FC	LV	BD	LH	VE	FC	VD	VE	VB	VC	LV	VD	VE	VB	LH	LH	LH	VE	VB	VC
VB	LH	LH	VC	VD	LH	VC	LV	FB	VD	LH	LH	VC	VD	VE	VD	LH	BE	VD	BB	FE	VB	BE	VD	VE
VD	VE	FD	BB	LH	LH	VE	LV	BD	LH	BB	LH	LH	VE	VD	LH	VE	VD	VE	LV	VB	VC	BD	FE	LV
FB	VD	VE	FC	FD	VE	BD	VC	VD	VE	VD	LH	VE	VD	BB	FE	VD	VE	LV	LV	LV	VB	VC	VB	VC
BD	FE	VD	LH	LH	VC	LV	VB	VE	FC	VB	VE	LV	VB	VC	VB	LH	VC	LV	LV	LV	VD	VE	VD	VE
LV	VB	LH	LH	LH	LH	VC	LV	LV	VB	VC	VD	VC	LV	VB	VC	VB	VE	FC	LV	VD	VE	FC	VB	VC
VD	BC	LH	LH	LH	LH	LH	VC	VD	VC	FD	LH	LH	VC	VD	LH	VC	VD	LH	BC	FE	VD	LH	BC	FE
